src/main/java/com/shippingcostestimator/enterprise/dto/ShippingCost.java

```diff
@@ -14,8 +14,8 @@
  */
 public @Data
 class ShippingCost {
-    int totalCostId;
-    int serviceLevel;
-    int rates;
-    int estArrival;
+    private int totalCostId;
+    private int serviceLevel;
+    private int rates;
+    private int estArrival;
```

Comment:
Added private to the destination and shipping cost class variables for best practices